lib/plugins/aws/customResources/resources/eventBridge/lib/utils.js

```diff
@@ -2,7 +2,13 @@
 
 function getEventBusName(eventBus) {
   if (eventBus && eventBus.startsWith('arn')) {
-    return eventBus.split('/').pop();
+    const splittedEventBus = eventBus.split('/');
+    // if it's a custom event bus
+    if (splittedEventBus.length === 1) {
```

Comment:
This will turn true only if there's no `/` in `eventBus` string. Is that a possible scenario?

Reply:
Good point. This should actually be `=== 2` since a custom event bus arn always includes only one `/`.